src/setfit/trainer.py

```diff
@@ -303,7 +297,7 @@ def hyperparameter_search(
         hp_space: Optional[Callable[["optuna.Trial"], Dict[str, float]]] = None,
         compute_objective: Optional[Callable[[Dict[str, float]], float]] = None,
         n_trials: int = 10,
-        direction: str = "minimize",
+        direction: str = "maximize",
```

Comment:
Thanks for updating this. Setfit's default metrics is accuracy, not the same as transformers default metric which is loss. So direction default value should be `maximize`.